www/src/pages/verticaldivider.mdx

```diff
@@ -0,0 +1,34 @@
+import {Stack, Placeholder, Box, VerticalDivider} from 'sassyboi'
+import Showcase from '../components/Showcase'
+import PropsTable from '../components/PropsTable'
+export const meta = {
+  name: 'VerticalDivider',
+  component: VerticalDivider,
+  description: 'A divider the separaters vertically aligned elements.',
+  inheritBoxProps: true,
+}
+
+
+<Showcase>
```

Comment:
The example here could be simplified - and also informs what the desired usage might look like instead.

One issue Im also seeing is that you are using `size` which actually sets width and height together on the underlying `Box`. Perhaps instead, VerticalDivider takes a prop called `size` of its own and internally applies it to the new `height` prop on the `Box`. If the prop is not given (is undefined) then it should be styled with `height: 100%` and possible a `min-height` or like 12px (using space props in css from theme). 

The new usage might look like this:

```
<Inline>
  <Placeholder height={40} width={40}/>

  <VerticalDivider />

  <Placeholder height={40} width={40}/>

  <VerticalDivider />

  <Placeholder height={40} width={40}/>
</Inline>
```

With that in place, then we can add a `dividers` prop to `Inline` (similar to stack) that inserts `VerticalDivider` between elements automatically. Lets discuss!

Reply:
Oooh good idea